Assets/Editor/ModelInspector.cs

```diff
@@ -677,22 +815,35 @@ static void ShowOtherInfo(Mesh mesh)
             EditorGUILayout.LabelField("Read/Write Enabled", mesh.isReadable.ToString());
             EditorGUI.indentLevel--;
         }
-
+        
         static void ShowBlendShapeInfo(Mesh mesh)
         {
             var blendShapeCount = mesh.blendShapeCount;
             if (blendShapeCount <= 0)
                 return;
-                
+
             EditorGUILayout.Space();
             EditorGUILayout.LabelField($"Blend Shapes: {blendShapeCount}", EditorStyles.boldLabel);
+            
+            var test = GUILayoutUtility.GetLastRect();
+            EditorGUILayout.BeginHorizontal();
+
+            var labelHeight = 25;
```

Comment:
`EditorGUIUtility.singleLineHeight`, but I'm actually wondering why it needs the size calculation at all. Normally you just make a scroll view, do whatever UI layout there, and it all magically works.